server/src/main/java/org/exoplatform/chat/services/mongodb/UserServiceImpl.java

```diff
@@ -493,6 +504,121 @@ private SpaceBean getSpace(String roomId, String dbName)
     return users;
   }
 
+  @Override
+  public List<UserBean> getSuggestionUsers(String currentUserName, String roomId, String filter, String dbName, int limit) {
```

Comment:
I am not sure to understand the add of the roomId here. We want to suggest users when adding members in a team, so why we would suggest someone already in the team ?
The spec mentions that "In Chat add-on, when creating a team, the autocompletion to add people should list my connections first (1) and then other users (2)."
I should miss something here :)